src/identity.js

```diff
@@ -1786,27 +1690,27 @@ export default function Identity(mpInstance) {
 
     this.sendUserIdentityChangeEvent = function(
         newUserIdentities,
-        method,
         mpid,
+
+        // Mislabeled, could be previous or incoming
         prevUserIdentities
     ) {
-        var currentUserInMemory, userIdentityChangeEvent;
-
-        if (!mpid) {
-            if (method !== Modify) {
-                return;
-            }
-        }
-
-        currentUserInMemory = this.IdentityAPI.getUser(mpid);
+        // What's the difference between all of our getuser methods?
+        // We may not need to reaquire the user since we know the mpid
+        // WE already have this user from the identityApiResult
```

Comment:
let's make a jira ticket and remove this comment since this is an official PR and not a draft now.